nb/projects/BarbourSmith/belt_sprockets/belt_sprockets.nb

```diff
@@ -289,9 +276,7 @@ export const T10 = {
     [2.971997, 0.007239, 0],
     [3.06511, 0, 0],
     [3.06511, -1, 0]
-  )
-    .md('T10 profile')
```

Comment:
Why did these go away?

Reply:
Ah, they're getting invoked indirectly when-ever you build these -- e.g., from examples -- so you'd get 20 copies of each for each example.

Generally you want these things at the example level rather than the reuse level.